requirements.txt

```diff
@@ -2,7 +2,7 @@ argon2-cffi==23.1.0
 argon2-cffi-bindings==21.2.0
 asgiref==3.8.1
 certifi==2024.7.4
-cffi==1.16.0
+cffi==1.17.0
 cryptography==43.0.0
 Django==5.1
```

Comment:
Confirmed that Django would be updated for resolving error on Trivy scan. ([Ref](https://github.com/hwakabh/bennu-official/pull/205#issuecomment-2306510565))